rules/xcodeproj.bzl

```diff
@@ -99,6 +99,12 @@ def _xcodeproj_aspect_impl(target, ctx):
     bazel_bin_subdir = "%s/%s" % (target.label.workspace_root, target.label.package)
 
     if AppleBundleInfo in target:
+        swift_objc_header_path = None
+        if SwiftInfo in target:
```

Comment:
this seems ick, and the iteration over all headers a potential performance concern: could we instead look at implementing https://github.com/bazelbuild/rules_swift/issues/291 ?

Reply:
Based upon that issue's conversation, it looks like the bazel team is thinking about returning a totally separate provider (called IDEInfo or something along those lines) and returning this type of information in there, as they don't think it should belong in the SwiftInfo. I'm not sure if it's worthwhile for me to try to implement that ticket right now, but I did leave a comment and +1 the issue.

I also looked at Lyft's workaround (https://github.com/lyft/envoy-mobile/pull/797/files) which is very similar to my approach here. In their case, they are pulling the headers list from the CCInfo provider.